File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/common/pool-work-flow.ts
import { PoolWorkFlowItemTypeEnum } from "projects/claimcare/src/app/claim-manager/shared/enums/pool-work-flow-item-type.enum";
import { WorkPoolEnum } from "../enums/work-pool-enum";

export class PoolWorkFlow {
    poolWorkFlowId: number;
    poolWorkFlowItemType: PoolWorkFlowItemTypeEnum;
    itemId: number;
    workPool: WorkPoolEnum;
    assignedByUserId: number;
    assignedToUserId: number | null;
    effectiveFrom: Date;
    effectiveTo: Date | null;
    instruction: string;
    isDeleted: boolean;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;
}

File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/PoolWorkFlow.cs

using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class PoolWorkFlow
    {
        public int PoolWorkFlowId { get; set; } // PoolWorkFlowId (Primary key)
        public PoolWorkFlowItemTypeEnum PoolWorkFlowItemType { get; set; }
        public int ItemId { get; set; } // ItemId
        public WorkPoolEnum WorkPool { get; set; } // WorkPoolId
        public int AssignedByUserId { get; set; } // AssignedByUserId
        public int? AssignedToUserId { get; set; } // AssignedToUserId
        public System.DateTime EffectiveFrom { get; set; } // EffectiveFrom
        public System.DateTime? EffectiveTo { get; set; } // EffectiveTo
        public string Instruction { get; set; } // Instruction
        public bool IsDeleted { get; set; } // isDeleted
        public string CreatedBy { get; set; } // CreatedBy (length: 50)
        public System.DateTime CreatedDate { get; set; } // CreatedDate
        public string ModifiedBy { get; set; } // ModifiedBy (length: 50)
        public System.DateTime ModifiedDate { get; set; } // ModifiedDate
    }
}